src/client/sandbox/event/listeners.ts

```diff
@@ -144,19 +144,19 @@ export default class Listeners extends EventEmitter {
         };
     }
 
-    private _createElementOverridedMethods (el: HTMLElement|Window|Document) {
+    createEventTargetOverriddenMethods () {
```

Comment:
Why is this method public?

Reply:
It is used [here](https://github.com/DevExpress/testcafe-hammerhead/pull/2203/files#diff-9185f923f93f18209c3f343f36ca2c54R628):
```ts
const overriddenMethods = this.listenersSandbox.createOverriddenMethods();
```